index.js

```diff
@@ -1,10 +1,44 @@
 const babel = require('babel-core');
+const fs = require('fs');
+const path = require('path');
 
-const babelOpts = {
-  plugins: ['transform-flow-strip-types', 'syntax-jsx']
+/**
+ * Remove duplicates from array
+ * @param  {array} items Input array
+ * @return {array} Output array with unique values
+ */
+const arrayUnique = (items) =>
+    items.filter((item, pos, self) => self.indexOf(item) === pos);
+
+/**
+ * Construct Babel plugins config object.
+ * Honours user's global Babel plugins config. and adds
+ * required Flowtype plugin if not already available.
+ *
+ * @return {Object}
+ */
+const buildBabelrcConfig = () => {
+  const defaultConfig = ['transform-flow-strip-types', 'syntax-jsx'];
+  const babelrcPath = path.normalize(path.join(__dirname, '..', '..', '.babelrc'));
+
+  if (! fs.existsSync(babelrcPath)) {
```

Comment:
It looks like you're making two calls to "fs" when likely only one is necessary. You can always try to read the path and if it doesn't exist, then handle it with this short circuit.